app/grandchallenge/archives/views.py

```diff
@@ -492,10 +492,13 @@ class ArchiveItemsList(
     row_template = "archives/archive_items_row.html"
     search_fields = [
         "pk",
-        "name",
+        "values__interface__title",
+        "values__value",
+        "values__image__name",
+        "values__file",
```

Comment:
For some reason it did not allow me to query `values__file__name`. Since the `file` returns the name as well thouh, I guess querying like this is fine, though I'd be interested to know why `name` is not an allowed look-up for a FileField.

Reply:
`__image` is a model, which has the Field `name`, so you can look that up in the db. `__file` is a Field on the values, so you cannot do a through lookup there.